### PR TITLE
US : The horny PR

### PR DESCRIPTION
**Pull request by BellezaEmporium**
I'm sorry for that one x)

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1377,8 +1381,8 @@ https://peopletv-samsungus-ingest.akamaized.net/playlist.m3u8
 https://content.uplynk.com/channel/98828f7707b84dc496472d5789143df2.m3u8
 #EXTINF:-1 tvg-id="PlantBasedNetwork.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/iwd9Jxb.png" group-title="",Plant Based Network (1080p)
 https://hls-cdn.tvstartup.net/barakyah-channel/live/pbtv/playlist.m3u8
-#EXTINF:-1 tvg-id="PlayboyTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/cQDAzDF.jpg" group-title="XXX",Playboy TV [Offline]
-http://187.188.147.165:8000/play/a00l/index.m3u8
+#EXTINF:-1 tvg-id="PlayboyLatinoamerica.us" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="XXX",Playboy Latinoamérica
+http://45.5.8.78:8000/play/a03b
```
Comment (Dum4G) [inline channels/us.m3u:1385]: streams black screen

File: channels/us.m3u [hunk 2]
```diff
@@ -1367,6 +1369,8 @@ https://cdn87.theus6tv.tk/locals/SanDiego/kpbs-dt2.m3u8
 https://cs.ebmcdn.net/eastbay-live-hs-1/apt/mp4:apt-world/playlist.m3u8
 #EXTINF:-1 tvg-id="PCCTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/9PEAPwY.png" group-title="Local",PCC-TV Pinellas County FL (480p)
 http://granicusliveus1-a.akamaihd.net/pinellas/G1551_004/playlist.m3u8
+#EXTINF:-1 tvg-id="PenthouseWorld.us" tvg-country="US" tvg-language="" tvg-logo="" group-title="XXX",Penthouse World
+http://45.5.8.78:8000/play/a02n
```
Comment (Dum4G) [inline channels/us.m3u:1373]: infinite load

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:1385]: I shall check.

File: channels/us.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:1373]: Ah. I'll delete it if it doesn't change after some time.

**Review by Dum4G — APPROVED** [review]

**Review by jkemp814 — APPROVED** [review]

**Pull request merged**